bobtemplates/plone/addon/setup.py.bob

```diff
@@ -46,7 +46,13 @@ setup(
     keywords='Python Plone',
     author='{{{ author.name }}}',
     author_email='{{{ author.email }}}',
-    url='https://pypi.python.org/pypi/{{{ package.dottedname }}}',
+    url='https://github.com/collective/{{{ package.dottedname }}}',
+    project_urls={
+        'PyPI': 'https://pypi.python.org/pypi/{{{ package.dottedname }}}',
+        'Source': 'https://github.com/collective/{{{ package.dottedname }}}',
+        'Tracker': 'https://github.com/collective/{{{ package.dottedname }}}/issues',
+        'Documentation': 'https://{{{ package.dottedname }}}.readthedocs.io/en/latest/',
```

Comment:
I would keep this commented, b/c AFAIK it is a rare case to have docs for a project there.